src/bin/ota_plus.rs

```diff
@@ -55,22 +59,30 @@ fn main() {
 
     let matches = parser().get_matches();
     let outcome = || -> Result<()> {
-        let root = matches.value_of("root")
+        let cache = matches.value_of("cache")
             .map(PathBuf::from)
             .or_else(|| env::home_dir().map(|path| path.join(".ota-plus")))
             .ok_or_else(|| ErrorKind::IllegalArgument("Missing `path`.".into()))?;
 
         match matches.subcommand() {
-            ("init", Some(sub)) => cmd_init(root, sub),
-            ("keygen", Some(sub)) => cmd_keygen(root, sub),
+            ("init", Some(sub)) => cmd_init(cache, sub),
```

Comment:
I discovered this pattern while on vacation and face palmed at the abomination I'd written previously. :(